### ISSUE TITLE
Chunks out of order

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord-dev/ts-android#1956]
[en-x-demo1_psa_text_reg.a.tstudio.zip](https://github.com/unfoldingWord-dev/ts-android/files/826063/en-x-demo1_psa_text_reg.a.tstudio.zip)
![screenshot_2017-03-07-17-38-30](https://cloud.githubusercontent.com/assets/8171759/23681603/3a68f5bc-0355-11e7-8e50-3400a2360145.png)

**Comment by neutrinog** [comment]
@cckozie I cannot reproduce this. Are you certain you are using that build?

**Comment by cckozie** [comment]
Yes, that build. No special steps. Open an existing or create a new Psalms project, both are messed up, even in read mode.
![screenshot_2017-03-07-19-32-47](https://cloud.githubusercontent.com/assets/8171759/23684467/a9860e9e-0364-11e7-88c8-7798907d878b.png)

**Comment by cckozie** [comment]
Interesting... I deleted both of the translationStudio folders and tried it again creating a new project and now it looks fine. Could there be something messed up in the index file or sources elsewhere from a previous build?

**Comment by cckozie** [comment]
More weirdness...
I went back to a clean environment with 9.0 installed. (see issue  #1967) 
I upgraded to travis build 1261, did some testing, and the out of sequence card was not a problem.
I now uninstalled 1261 and upgraded to travis build 1268. Chunk 6 is not out of order again.
![screenshot_2017-03-09-16-17-30](https://cloud.githubusercontent.com/assets/8171759/23771392/5ca5bf4c-04dc-11e7-9cbc-67cdd6574f22.png)

**Comment by neutrinog** [comment]
ok there was a bug in the updater so the 00 chunk was not getting correctly renamed. This has been fixed.
